cmd/auth/internal/application/eventhandler/register.go

```diff
@@ -25,10 +25,16 @@ func Register(conn *grpc.ClientConn, eventBus eventbus.EventBus, topicToHandlerM
 
 			for topic, handler := range topicToHandlerMap {
 				// Will resubscribe to handler on error infinitely
-				go gollback.New(context.Background()).Retry(0, func(ctx context.Context) (interface{}, error) {
-					err := eventBus.Subscribe(ctx, topic, handler)
-					return nil, errors.Newf(errors.INTERNAL, "EventHandler %s unsubscribed (%v)", topic, err)
-				})
+				go func(topic string, handler eventbus.EventHandler) {
+					_, err := gollback.New(context.Background()).Retry(0, func(ctx context.Context) (interface{}, error) {
```

Comment:
Remove `_, err :=` and panic below, it is useless as `.Retry(0` will never return
> Will resubscribe to handler on error infinitely

Reply:
We can invert condition as well so it is more readable
```suggestion
	// Will retry infinitely until timeouts by context
	_, err := gollback.New(ctx).Retry(0, func(ctx context.Context) (interface{}, error) {
		if !grpc_utils.IsConnectionServing(connName, conn) {
			return nil, errors.Newf(" %s gRPC connection is not serving", connName)
		}

		for topic, handler := range topicToHandlerMap {
			// Will resubscribe to handler on error infinitely
			go func(topic string, handler eventbus.EventHandler) {
				gollback.New(context.Background()).Retry(0, func(ctx context.Context) (interface{}, error) {
					err := eventBus.Subscribe(ctx, topic, handler)

					return nil, errors.Newf(errors.INTERNAL, "EventHandler %s unsubscribed (%v)", topic, err)
				})
			}(topic, handler)
		}

		return nil, nil
	})
```